### PR TITLE
Add ROCK for pmmlserver

### PR DESCRIPTION
**Pull request by misohu**
ROCK for pmmlserver

Dockerfile: https://github.com/kserve/kserve/blob/master/python/pmml.Dockerfile

### CODE REVIEW

File: pmmlserver/rockcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    environment:
+      PYTHONPATH: /pmmlserver:/kserve
+
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+      dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
```
Comment (i-chvets) [inline pmmlserver/rockcraft.yaml:27]: For 22.04 we can have a better way of collecting packages (from our best practices):
`dpkg-query --root=${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/ -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W)`

**Review by SamirPS — APPROVED** [review]

**Comment by SamirPS** [comment]
i also think, you can use the base: bare

**Comment by misohu** [comment]
Merging as because of overlay I cannot use bare

**Comment by cjdcordeiro** [comment]
Nice work. I see it is already merged, but in any case, for future improvement, I'd recommend:
 - investigating if you really need java (it seems odd for a Python application)
 - if you need java, do you need jdk? I'm assuming you're not compiling anything at runtime, so jre should be enough?
 - if jre is enough, then I'd recommend installing it via chisel slices instead of the deb
 - if you can chisel it, then you don't need `overlay-packages` anymore, and thus you can try using `base: bare`

**Pull request merged**